src/clusterfuzz/_internal/bot/tasks/utasks/progression_task.py

```diff
@@ -291,86 +425,86 @@ def find_fixed_range(testcase_id, job_type):
-    # to chromecrash/.
     state = result.get_symbolized_data()
-    crash_uploader.save_crash_info_if_needed(testcase_id, max_revision,
-                                             job_type, state.crash_type,
-                                             state.crash_address, state.frames)
-    return
+    testcase.last_tested_crash_stacktrace = data_handler.filter_stacktrace(
+        stacktrace)
 
+    error_message = f'Still crashes on latest revision r{max_revision}.'
+    serialized_crash_stack_frames = crash_uploader.get_symbolized_stack_bytes(
+        state.crash_type, state.crash_address, state.frames)
+    progression_task_output = uworker_io.ProgressionTaskOutput(
+        crash_on_latest=True,
+        serialized_crash_stack_frames=serialized_crash_stack_frames,
+        crash_revision=max_revision)
+    return uworker_io.UworkerOutput(
+        testcase=testcase,
+        progression_task_output=progression_task_output,
+        error_message=error_message)
+
+  # TODO(alhijazi): Check if these can be removed as they don't seem to be used
+  # anywhere.
   if result.unexpected_crash:
     testcase.set_metadata('crashes_on_unexpected_state', True)
   else:
     testcase.delete_metadata('crashes_on_unexpected_state')
 
   # Verify that we do crash in the min revision. This is assumed to be true
   # while we are doing the bisect.
-  result = _testcase_reproduces_in_revision(testcase, testcase_file_path,
-                                            job_type, min_revision)
-  if result and not result.is_crash():
-    testcase = data_handler.get_testcase_by_id(testcase_id)
+  result, error = _testcase_reproduces_in_revision(testcase, testcase_file_path,
+                                                   job_type, min_revision)
+  if error is not None:
+    return error
 
-    # Retry once on another bot to confirm our result.
-    if data_handler.is_first_retry_for_task(testcase, reset_after_retry=True):
-      tasks.add_task('progression', testcase_id, job_type)
-      error_message = (
-          'Known crash revision %d did not crash, will retry on another bot to '
-          'confirm result' % known_crash_revision)
-      data_handler.update_testcase_comment(testcase, data_types.TaskState.ERROR,
-                                           error_message)
-      data_handler.update_progression_completion_metadata(
-          testcase, max_revision)
-      return
-
-    data_handler.clear_progression_pending(testcase)
+  if result and not result.is_crash():  # pylint: disable=no-member
     error_message = (
-        'Known crash revision %d did not crash' % known_crash_revision)
-    data_handler.update_testcase_comment(testcase, data_types.TaskState.ERROR,
-                                         error_message)
-    task_creation.mark_unreproducible_if_flaky(testcase, True)
-    return
-
+        f'Known crash revision {known_crash_revision} did not crash')
```

Comment:
I wouldn't bother with error_message here it doesn't add much, can you just repeat that string twice in the error handler?

Reply:
I'm not sure I understand what string to repeat and where?